python/GafferSceneUI/_InspectorColumn.py

```diff
@@ -420,11 +420,85 @@ def __keyPress( column, pathListing, event ) :
 
 	return False
 
+def __dragEnter( path, pathListing, event ) :
+
+	column = pathListing.columnAt( event.line.p0 )
```

Comment:
Having to re-find the column like this seems awkward. `KeySignal` passes the column as the first argument, so perhaps we could do the same for the drag signals? That would be consistent with most other Widget and Gadget signals (but not the ButtonSignals on PathColumn, which should be brought into line at some point).

Reply:
Good idea! Adjusted the drag signals to also pass the column in 5a4b8a66bb4311a801d54de2282c77519c0176dd, bac7e5ff295f649e54e3181cb6f443a1b7fb96a8, and 17e048bc0367cbdde25c6d8fb5da0d5ccee0c9b6.